### DISCUSSION TITLE
Can't figure out my water meter

### DISCUSSION THREAD

**Discussion by MitchTalmadge** [allangood/rtlamr2mqtt#228] [General]
I am trying to add my water meter (had success with gas and electric) but I am not finding any readings from it so far. I'm going to log my notes here in case it becomes useful to anyone else.

First, here's some photos of the meter, which lives inside of a deep hole with a lid on top. The meter has a wire running to the lid, which has an external antenna. The meter is from "Badger Meter", model "Absolute Digital Encoder (ADE) 64501-001" and the antenna is from "Itron", model "CFG-0900-003". 

![image](https://github.com/allangood/rtlamr2mqtt/assets/8848080/af636afb-8f0d-4e8e-9f54-20fd2bcc7e87)

![image](https://github.com/allangood/rtlamr2mqtt/assets/8848080/6808c5ae-bfac-49f3-b2b7-261d6d0c44b0)

There is an ID on the meter but it is not in the logs:

![image](https://github.com/allangood/rtlamr2mqtt/assets/8848080/70064fc3-e8cd-46db-8159-df11eab96a32)

The underside of the lid has this ID which is also not in the logs:

![image](https://github.com/allangood/rtlamr2mqtt/assets/8848080/d9194cf8-2a34-4f86-ad47-30fdaed18a89)

For the meter, I believe these are the instructions: 
[ENC-UM-01061-EN-05_Model ADE Absolute Digital Encoder Manual.pdf](https://github.com/allangood/rtlamr2mqtt/files/14016871/ENC-UM-01061-EN-05_Model.ADE.Absolute.Digital.Encoder.Manual.pdf)

And for the lid antenna, I found this, which mentions the lid antenna: 
[100W TTL.pdf](https://github.com/allangood/rtlamr2mqtt/files/14016904/100W.TTL.pdf)

Note that I only have the lid, not the transceiver that they have in that document:

![image](https://github.com/allangood/rtlamr2mqtt/assets/8848080/bf30fd50-dde7-412b-9b41-324f47760791)

Anyway I am not sure what frequency this operates on. Maybe I need to run another USB SDR device on another freq? I'll post here if I find more info

**Comment by MitchTalmadge** [comment]
I found my water meter [on eBay](https://www.ebay.com/itm/285247949329), so I have a cleaner picture to share. 

![image](https://github.com/allangood/rtlamr2mqtt/assets/8848080/09231664-9daa-4338-9980-725e7c1be3e2)

**Reply by MitchTalmadge** [reply]
Found some more documentation on this meter

[02o-ADE-REGISTER.pdf](https://github.com/allangood/rtlamr2mqtt/files/14017224/02o-ADE-REGISTER.pdf)

**Reply by MitchTalmadge** [reply]
I did not see any Orion endpoints connected to the water meter, but I'll look again tomorrow. If it's using Orion, then it will be frequency hopping anywhere between 902 & 928 MHz as defined by this document:

[ORION Water Endpoints ME Endpoint Product Data Sheet (ORI-DS-00655-EN).pdf](https://github.com/allangood/rtlamr2mqtt/files/14017294/ORION.Water.Endpoints.ME.Endpoint.Product.Data.Sheet.ORI-DS-00655-EN.pdf)

So maybe I can get readings by changing my center frequency and some other things... Gotta look into it.

**Reply by justinyounger** [reply]
@MitchTalmadge Did you ever figure this out? I have the same one and not seeing its ID populate, or not seeing a clearly matching ID.

**Reply by MitchTalmadge** [reply]
@justinyounger unfortunately I have not :( but I also haven't spent much time trying. I would definitely like to get it working this year before sprinklers start again.

**Reply by justinyounger** [reply]
@MitchTalmadge Let me know if you do, I am in Chicago and can see a bunch of ID's registering, but none of them seem to match what I see on my device. I am currently trying to match the consumption, but it doesnt seem to match...the visual amount? Ill let you know if I ever figure it out too.
